modules/server/src/main/scala/seqexec/server/tcs/TcsControllerEpicsCommon.scala

```diff
@@ -28,6 +28,8 @@ import seqexec.server.EpicsCodex.encode
 import seqexec.server.EpicsCommand
 import seqexec.server.SeqexecFailure
 import seqexec.server.tcs.TcsController._
+import squants.Length
```

Comment:
Unrelated but we are moving towards using coulomb rather than squants. Maybe it should be changed at least on observe